### ISSUE TITLE
Evaluate pretrained model on validation and testing images using keras library

### ISSUE THREAD

**Issue by SalemAmeen** [heuritech/convnets-keras#9]
Sorry this is not an issue 

Any idea of how once can evaluate convnets.py on all of the validation and testing images saved in my machine. I know in keras there is an evaluation function but I could not make it work.

im = preprocess_image_batch(['examples/dog.jpg'],img_size=(256,256), crop_size=(227,227), color_mode="rgb")
sgd = SGD(lr=0.1, decay=1e-6, momentum=0.9, nesterov=True)
model = convnet('alexnet',weights_path="weights/alexnet_weights.h5", heatmap=False)
model.compile(optimizer=sgd, loss='mse')
out = model.predict(im)

Now I want to evaluate the model on the validation image but I could not define x_test and y_test.

score = model.evaluate(x_test,y_test,verbose=0)
